CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ This projects adheres to [Semantic Versioning](http://semver.org/) and [Keep a C
 
 _Nothing yet._
 
+
```

Comment:
0.14.1 is already released, so not sure why this is part of this PR?

Reply:
The original sniff was based on `master` and pulled against `develop`. @moorscode has rebased the PR now against `develop`, so this should be sorted.